### PR TITLE
added trailing forward slashes

### PR DESCRIPTION
**Pull request by Ankitkumar94**

### CODE REVIEW

File: app/index.js [hunk 1]
```diff
@@ -45,9 +45,9 @@ module.exports = generators.Base.extend({
   prompting: function() {
     this.getLocalDirDefault = function(answers) {
       if (answers.deployType === 'sdk') {
-        return os.homedir().replace(/\\/g,'/') + '/' + DEFAULT_SDK_SERVER + '/' + DEFAULT_OWA_DIR;;
+        return os.homedir().replace(/\\/g,'/') + '/' + DEFAULT_SDK_SERVER + '/' + DEFAULT_OWA_DIR + '/';;
       } else if (answers.deployType === 'standalone') {
-        return os.homedir().replace(/\\/g,'/') + '/' + OPENMRS_STANDALONE_APPDATA_DIR + '/' + DEFAULT_OWA_DIR;;
+        return os.homedir().replace(/\\/g,'/') + '/' + OPENMRS_STANDALONE_APPDATA_DIR + '/' + DEFAULT_OWA_DIR + '/';;
       } else {
         return DEFAULT_ENTERPRISE_APPDATA_DIR + '/' + DEFAULT_OWA_DIR; // There's some work to do here to create a sane default for Windows
```
Comment (psbrandt) [inline app/index.js]: What about this one?

File: app/index.js [hunk 2]
```diff
@@ -191,7 +191,7 @@ module.exports = generators.Base.extend({
           version: this.pkg.version,
           appId: this.appId,
           appEntryPoint: this.appEntryPoint,
-          localDeployDirectory: this.localDeployDirectory
+          localDeployDirectory: this.localDeployDirectory.endsWith('/') ? this.localDeployDirectory : this.localDeployDirectory + '/'
```
Comment (psbrandt) [inline app/index.js]: If we have this, do we need to repeat it in the `getLocalDirDefault()` function?

File: app/index.js (hunk 1 above)
Comment (Ankitkumar94) [inline app/index.js]: This indicates the thing which is the suggested local directory i.e `<C:/Users/AK/openmrs-standalone-2.4/appdata/owa/>`

Comment (psbrandt) [inline app/index.js]: If I use `DEFAULT_ENTERPRISE_APPDATA_DIR`, do I run the risk of a failed deploy due to no trailing `/`?

File: app/index.js (hunk 2 above)
Comment (Ankitkumar94) [inline app/index.js]: And this checks after adding the path that the path contains "/" or not.
if user uses the default, then "/" will automatically be there after adding the above code and if user enters the path manually and forgets to enter trailing "/", then this code will automatically add it

Comment (psbrandt) [inline app/index.js]: Sure, but do we need _both_ this check _and_ the checks above (in the `getLocalDirDefault()` function)?

Comment (Ankitkumar94) [inline app/index.js]: oh oh  I got what you want to say !! 
sorry, actually mind was somewhere else !
updating it right now

**Comment by Ankitkumar94** [comment]
Done :)

**Pull request merged**